pages/contact.md

```diff
@@ -1 +1,5 @@
-If you'd like to get in touch please email us at [hello@exercism.io](mailto:hello@exercism.io).
+The best place to get help with Exercism is on Github at [https://github.com/exercism/exercism](https://github.com/exercism/exercism). You are welcome to open issues for support requests, ask questions, suggest features, or anything else there.
+
+If you'd like to get in touch privately please email us at [hello@exercism.io](mailto:hello@exercism.io), but please note that due to the quantity of emails we get we may not reply if the question would be more appropriate on GitHub. 
+
+If you have a Code of Conduct violation to report, please email us at [hello@exercism.io](mailto:hello@exercism.io) and we will deal with that as a priority.
```

Comment:
Should there be a blurb here asking that `code of conduct` appear in the subject line, since the same email address is being used for everything and because it is stated above that some emails may not get a response.

Reply:
This seems like a good addition. The rest looks good!

Reply:
Good points. Thanks.